app/src/pages/UploadDocumentsPage/index.tsx

```diff
@@ -10,6 +10,7 @@ import {
 } from "../../types/pages/UploadDocumentsPage/types";
 import uploadDocument from "../../helpers/requests/uploadDocument";
 import { useBaseAPIUrl } from "../../providers/configProvider/ConfigProvider";
+import {PatientDetails} from "../../types/components/types";
```

Comment:
We've imported here but we don't seem to use it anywhere, should this be removed?